modules/sbo-projecting-the-binding-data.adoc

```diff
@@ -0,0 +1,17 @@
+[id="sbo-projecting-the-binding-data_{context}"]
+= Projecting the binding data
+
+Depending on your workload requirements and environment, you can choose to project the binding data either as files or environment variables.
```

Comment:
```suggestion
Depending on your workload requirements and environment, you can project the binding data as files or environment variables.
```

Reply:
Please see my response above.

Reply:
How about something like "Depending on your workload requirements and environment, you can keep the default configuration, which projects the binding data as files, or you can configure the XYZ to project the binding data as environment variables."